dumpgenerator.py

```diff
@@ -530,7 +532,9 @@ def getXMLPageCore(headers={}, params={}, config={}, session=None):
             handleStatusCode(r)
             xml = fixBOM(r)
         except requests.exceptions.ConnectionError as e:
-            raise ExportAbortedError(config['index'])
+            print '    Connection error {0}'.format(e[0])
+            if abortonconnectionerror:
```

Comment:
I believe this will always return False since there is no code that changes the state of `abortonconnectionerror`. Also, use CamelCase if possible.